tests/integration/test_fim/test_files/test_audit_buffer/data/test_cases/cases_audit_buffer_over_time_no_overflow.yaml

```diff
@@ -0,0 +1,12 @@
+- name: Audit Buffer full - all files detected in whodata
+  description: Verify if files are added below the queue's size, after space is freed, all files appear in whodata mode
+  configuration_parameters:
+    TEST_DIRECTORIES: /
+    QUEUE_SIZE: 50
+    MAX_EPS: 5
+    FREQUENCY: 20
+  metadata:
+    files_first_insert: 50      # Fills queue 100%  - log is only generated when it overflows.
+    wait_time: 5                # Time to wait between inserts to allow for files to be  processed
```

Comment:
```suggestion
    wait_time: 5                # Time to wait between inserts to allow for files to be processed
```

Reply:
done in https://github.com/wazuh/wazuh-qa/pull/3990/commits/7e1dbb3b46c199abda25f292b2b1e0753d9a42b0